theano/tensor/basic.py

```diff
@@ -2298,12 +2298,15 @@ def second(a, b):
 
 
 @constructor
-def ones_like(model, dtype=None):
+def ones_like(model, dtype=None, opt=False):
     """equivalent of numpy.ones_like
     Parameters
     ----------
     model : tensor
     dtype : data-type, optional
+    opt : If True, we will return a constant instead of a graph when possible.
```

Comment:
Currently, this only applies when model is a tensor.scalar, right? If so, we should probably say that.